src/sentry/static/sentry/app/components/autoComplete.jsx

```diff
@@ -221,11 +221,11 @@ class AutoComplete extends React.Component {
   };
 
   moveHighlightedIndex = (step, e) => {
-    const listSize = this.items.size - 1;
     let newIndex = this.state.highlightedIndex + step;
+    const listSize = this.itemCount || this.items.size;
```

Comment:
When would `this.itemCount` be different to `this.items.size`?

Reply:
Let's add a comment here addressing this.

The answer is that `this.items` only contain the rows that are rendered so when they are virtualized you only get a subset of the items. `itemCount` are all rows (except for groupLabels)